lib/cylc/job_file.py

```diff
@@ -178,17 +184,17 @@ def _write_err_trap(cls, handle, job_conf):
     for S in $VACATION_SIGNALS $FAIL_SIGNALS; do
         trap "" $S
     done
-    if [[ -n ${CYLC_TASK_LOG_ROOT:-} && -f $CYLC_TASK_LOG_ROOT.status ]]; then
-        rm -f $CYLC_TASK_LOG_ROOT.status
```

Comment:
Yes. It should then contain the full history. (The removal of the status file on vacation has really become a bug since #1166 + #1183 change, as it would remove the entries for the batch system name, etc.)